language/de_x_sie/cli.php

```diff
@@ -69,21 +69,30 @@
-	'CLI_DESCRIPTION_REPARSER_REPARSE'			=> 'Verarbeitet gespeicherte Texte mit dem aktuellen text_formatter-Dienst neu.',
-	'CLI_DESCRIPTION_REPARSER_REPARSE_ARG_1'	=> 'Text-Art die neu verarbeitet werden soll. Leer lassen, um alles neu zu verarbeiten.',
+	
+	'CLI_DESCRIPTION_REPARSER_LIST'						=> 'Listet die Arten von Texten auf, die neu verarbeitet werden können.',
+	'CLI_DESCRIPTION_REPARSER_REPARSE'					=> 'Verarbeitet gespeicherte Texte mit dem aktuellen text_formatter-Dienst neu.',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_ARG_1'			=> 'Text-Art die neu verarbeitet werden soll. Leer lassen, um alles neu zu verarbeiten.',
 	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_DRY_RUN'		=> 'Keine Änderungen speichern; nur ausgeben, was passieren würde',
 	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RANGE_MIN'	=> 'Niedrigste Eintrags-ID, die verarbeitet wird',
 	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RANGE_MAX'	=> 'Höchste Eintrags-ID, die verarbeitet wird',
 	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RANGE_SIZE'	=> 'Geschätzte Zahl von Einträgen, die in einem Durchgang verarbeitet werden',
 	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_RESUME'		=> 'Startet die Verarbeitung dort, wo die letzte Ausführung endete',
-	'CLI_DESCRIPTION_RECALCULATE_EMAIL_HASH'	=> 'Berechnet die Einträge im Feld user_email_hash der Benutzer-Tabelle neu.',
-	'CLI_DESCRIPTION_SET_ATOMIC_CONFIG'			=> 'Legt den Wert für eine Konfigurations-Einstellung fest, wenn die derzeitige Einstellung dem angegebenen Vergleichswert entspricht',
-	'CLI_DESCRIPTION_SET_CONFIG'				=> 'Legt den Wert für eine Konfigurations-Einstellung fest',
+	'CLI_DESCRIPTION_RECALCULATE_EMAIL_HASH'			=> 'Berechnet die Einträge im Feld user_email_hash der Benutzer-Tabelle neu.',
 
-	'CLI_DESCRIPTION_THUMBNAIL_DELETE'		=> 'Löscht alle existierenden Vorschaubilder.',
-	'CLI_DESCRIPTION_THUMBNAIL_GENERATE'	=> 'Erstellt alle fehlenden Vorschaubilder.',
-	'CLI_DESCRIPTION_THUMBNAIL_RECREATE'	=> 'Erstellt alle Vorschaubilder neu.',
+	'CLI_DESCRIPTION_SET_ATOMIC_CONFIG'					=> 'Legt den Wert für eine Konfigurations-Einstellung fest, wenn die derzeitige Einstellung dem angegebenen Vergleichswert entspricht',
+	'CLI_DESCRIPTION_SET_CONFIG'						=> 'Legt den Wert für eine Konfigurations-Einstellung fest',
+
+	'CLI_DESCRIPTION_THUMBNAIL_DELETE'					=> 'Löscht alle existierenden Vorschaubilder.',
+	'CLI_DESCRIPTION_THUMBNAIL_GENERATE'				=> 'Erstellt alle fehlenden Vorschaubilder.',
+	'CLI_DESCRIPTION_THUMBNAIL_RECREATE'				=> 'Erstellt alle Vorschaubilder neu.',
+
+	'CLI_DESCRIPTION_UPDATE_CHECK'					=> 'Prüft, ob das Board auf dem neusten Stand ist.',
+	'CLI_DESCRIPTION_UPDATE_CHECK_ARGUMENT_1'		=> 'Name der zu überprüfenden Erweiterung (bei „all“, werden alle Erweiterungen überprüft)',
+	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_CACHE'		=> 'Führe den „check“-Befehl mit Cache aus.',
+	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_STABILITY'	=> 'Führe den „check“-Befehl mit der Auswahl „stable“- oder „unstable“-Versionen aus.',
```

Comment:
Ich glaube die Formulierung passt nicht zu unseren bisherigen Beschreibungen. "Führe den Befehl aus" beschreibt nicht direkt den Befehl. Besser passt imho "Führt den Befehl aus", also z.B. "Führt den 'check' Befehl mit Cache aus". Das betrifft auch die "du" Version.

So oder so wäre das in der Form auch nicht für de_x_sie tauglich, dafür müsste es sonst "Führen Sie den Befehl aus" heißen.